main/templates/main/help.html

```diff
@@ -0,0 +1,8 @@
+{% extends "main/base.html" %}
+{% block title %}
+  Help
+{% endblock title %}
```

Comment:
I've seen it in other pages - I think the process manager and controller do the same - but why do we have a title block if we are then adding a `h1` header within the content block?

Reply:
This was just something I was playing around with, wanted to see the difference between the two.

Reply:
The title block doesn't add content to the page it sets the content of the [title tag](https://www.w3schools.com/tags/tag_title.asp) within the html header - https://github.com/ImperialCollegeLondon/drunc_ui/blob/33b8433fabd19ccf6ae7e2f05c2f674463b5dbc0/main/templates/main/base.html#L6-L10. This effectively sets the name of the tab in the browser.